requirements-dev.txt

```diff
@@ -15,4 +15,5 @@ clang-format==13.0.0
 mypy>=0.971
 types-protobuf==3.18.4
 black>=22.3
-isort[colors]>=5.10
+isort>=5.10.1
+colorama>=0.4.5
```

Comment:
To make `requirements-dev.txt`compact, if it only fails with Python 3.7 somehow while using isort, could we add a TODO here saying colorama can be removed after Python 3.7 is deprecated?

It seems to me at least the latest isort does set colorama as a required dependency: https://github.com/PyCQA/isort/blob/12cc5fbd67eebf92eb2213b03c07b138ae1fb448/pyproject.toml#L47.

Reply:
looking at the pyproject, colorama is set as an optional dependency. usually installing `isort[colors]` will work (which downloads colorama). In this case something's wrong with the version resolution.

Do we actually see prolonged test runs in onnx for py37? It could also be that onnx-script didn't have `wheels` in the environment before.

Reply:
Even though we don't really see the prolonged test runs for py37, it's potentially a risk proved in another repo, and now that we already have this PR, I think it's no harm to merge? But I am fine either way.